packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -99,17 +105,20 @@ export class Outbox {
 	private maybeFlushPartialBatch() {
 		const mainBatchSeqNums = this.mainBatch.sequenceNumbers;
 		const attachFlowBatchSeqNums = this.attachFlowBatch.sequenceNumbers;
+		const blobAttachSeqNums = this.blobAttachBatch.sequenceNumbers;
```

Comment:
Looked into what we discussed (around adjusting the `BlobManager` flow). I see a world in which it's possible to handle normal flow and reconnect flow with the model we have in `BlobManager`, but it will take a bit of refactoring (as we currently send `"blobAttach"` ops to pending state manager when we transition to offline). The `"blobAttach"` ops submitted while offline have an empty storage Id (which is filled upon reconnecting and resubmitting the op).

My thinking on what we should probably do:
- async upload the blob
- async wait for current batch to finish and then send `"blobAttach"` op
    - if runtime is no connected, send nothing
- return result (handle with local blobId)

For reconnect:
- async re-upload the blob (if not already sent)
- async wait for current batch to finish and then send `"blobAttach"` op
    - might need to be careful about reconnects here
- resubmit other ops